### DISCUSSION TITLE
MPI parallel processing

### DISCUSSION THREAD

**Discussion by super-J123** [byuflowlab/FLOWVPM.jl#9] [Q&A]
“Single-node CPU parallel processing through OpenMPI”
I don't find mpi in the code，what is the default number of cores used by VPM? 
How should I change the number of cores used by VPM?

thank u very much！！

**Comment by EdoAlvarezR** [comment]
Parallelization is done inside [the FLOWExaFMM package](https://github.com/byuflowlab/FLOWExaFMM.jl). The number of cores is automatically set during compilation of that package.
